UI/PreferencesUI/UIxPreferences.m

```diff
@@ -259,7 +260,16 @@ - (NSArray *) passwordPolicy
   NSObject <SOGoSource> *userSource;
 
   userSource = [user authenticationSource];
-  return [userSource userPasswordPolicy];
+  NSMutableDictionary *translations = [[NSMutableDictionary alloc] init];
```

Comment:
Keep variables declarations at the top of the current block (in this case, at the beginning of the method).

Reply:
`translations` var is now on the top of the method, below `userSource`

Reply:
Please do the same with `policy` and `translatedUserPolicy`. Thanks!

Reply:
done ;)